hicexplorer/hicSumMatrices.py

```diff
@@ -6,10 +6,10 @@
 
 
 def parse_arguments(args=None):
-    
+
     parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter,
                                      description=('Adds Hi-C matrices of the same size. Format '
-                                                  'has to be .npz.'))
+                                                  'has to be .h5.'))
```

Comment:
I think it could be on any of the formats recognized by HiCexplorer but the output has to be .h5